ironfish-cli/src/commands/wallet/mint.ts

```diff
@@ -158,26 +163,47 @@ export class Mint extends IronfishCommand {
       )
     }
 
+    const mintResponse = await client.mintAsset({
+      account,
+      assetId,
+      fee: CurrencyUtils.encode(fee),
+      metadata,
+      name,
+      value: CurrencyUtils.encode(amount),
+      confirmations,
+    })
+
+    const rawTransactionBytes = Buffer.from(mintResponse.content.transaction, 'hex')
+    const rawTransaction = RawTransactionSerde.deserialize(rawTransactionBytes)
+
     if (!flags.confirm) {
       const nameString = name ? `Name: ${name}` : ''
       const metadataString = metadata ? `Metadata: ${metadata}` : ''
       const includeTicker = !!assetId
       const amountString = CurrencyUtils.renderIron(amount, includeTicker, assetId)
-      const feeString = CurrencyUtils.renderIron(fee, true)
+      const feeString = CurrencyUtils.renderIron(rawTransaction.fee, true)
       this.log(`
 You are about to mint ${nameString} ${metadataString}
```

Comment:
I think we could probably get rid of this entire message and the confirmation when we pass in the `--rawTransaction` flag. It still makes it seem like it it's going to get minted even though it doesn't say `* This action is NOT reversible *`. Makes more sense to me if it just spit out the raw transaction after it received all the inputs

Reply:
I want to keep printing the details of the transaction with `rawTransaction` flag, because user still want to see the details of the transaction. In `wallet:post`, they wont be able to see as much details before proceed with posting.